### ISSUE TITLE
/edituser not found

### ISSUE THREAD

**Issue by t3chguy** [ProjectFirrre/zncstrap#20]
whenever I'm on the Home page of the ZNC Panel (after auth)
and I click on my username for the dropdown, and then click Edit Settings
it redirects me to /edituser which results in a 404.

**Comment by lwatsondev** [comment]
I had the same issue and fixed it by using an nginx location rule, since I have nginx acting as a reverse proxy to ZNC's webadmin.

```
location = /edituser {
    return 301 /mods/global/webadmin/edituser;
}
```

**Comment by ItsLukeJames** [comment]
Same issue here

**Comment by itslukej** [comment]
I've made a pull request to fix the error, see #24
